frontend/src/components/books/BookDetails/BookImage/BookImage.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import "jest-styled-components";
```

Comment:
Could this be imported in `setupTests.ts`?

Reply:
Good point, will do so! 👍🏼